pgfinder/matching.py

```diff
@@ -270,7 +243,7 @@ def clean_up(ftrs_df: pd.DataFrame, mass_to_clean: Decimal, time_delta: float) -
         # Get retention time value from row
         rt = row.rt
         # Get theoretical monoisotopic mass value from row as list of values
-        intact_mw = list(str(row.theo_mwMonoisotopic).split(","))
+        intact_mw = str(row.theo_mwMonoisotopic)
```

Comment:
Why is this coerced  into a string and then a decimal (at line 264), please?

Reply:
Looks like it's to avoid some awful floating-point imprecision?
![image](https://user-images.githubusercontent.com/6251883/210016843-c6b49050-f8cd-4b6c-8b70-d6efaac79234.png)

With that being said, the call to `quantize` on 264 rounds that all away anyways! I've gotten rid of the `str` conversion!

Reply:
Well tracked down. Happy that it's gone. A future refactor should help with the typing / fp issues.